frontend/src/app/modules/grids/widgets/news/news.component.spec.ts

```diff
@@ -8,11 +8,12 @@ import { States } from 'core-app/components/states.service';
 import { UserDmService } from 'core-app/modules/hal/dm-services/user-dm.service';
 import { HalResourceService } from "core-app/modules/hal/services/hal-resource.service";
 import { HttpClientModule } from "@angular/common/http";
+import { By } from '@angular/platform-browser';
 
-describe('shows news', () => {
+fdescribe('shows news', () => {
```

Comment:
You need to remove the `fdescribe` in this line. It results in only this spec being tested (and will fail on our CI)

Reply:
@oliverguenther i correct this part,